react/index.js

```diff
@@ -318,8 +306,8 @@ const withLinkStateMinicartQuery = graphql(fullMinicartQuery, {
   options: () => ({ ssr: false }),
   props: ({ data: { minicart } }) => ({
     linkState: {
-      minicartItems: minicart && minicart.items,
-      orderForm: minicart && minicart.orderForm,
+      minicartItems: minicart && JSON.parse(minicart.items),
+      orderForm: minicart && JSON.parse(minicart.orderForm),
```

Comment:
are you absolutely sure these values are going to be valid JSON strings? `JSON.parse` throws errors when it fails

Reply:
I am.